components/form/FCreditCardInput.vue

```diff
@@ -22,8 +22,8 @@ FField.FCreditCardInput(
     :rules="[() => isValid]"
     hide-error-icon
     hide-hint
-    :mask="creditCard?.mask"
-    @focus="handleFocus"
+    :mask="creditCard?.mask || '#### #### #### ####'"
```

Comment:
plutôt un nulish coalescing ici, et une variable `const defaultMask = '#### #### #### ####'` dans le setup
```suggestion
    :mask="creditCard?.mask ?? defaultMask"
```